### PR TITLE
MAS 2.3.00 MAG SDK Sample app changes

### PR DESCRIPTION
**Pull request by sr043642**
Android App Changes: Sample apps are updated with SDK 2.3.00 and Android 13, A new Sample MAGCertificatePinning has been added
IOS App Changes: Sample apps are updated with SDK 2.3.00

### CODE REVIEW

File: MAG_SDK_Mobile_Apps/Android/Sample-App-MAS-Android-Access-API-Geolocation-And-OTP/MASAccessAPISample/build.gradle [hunk 1]
```diff
@@ -7,25 +7,25 @@
  */
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
```
Comment (zakirjt) [inline MAG_SDK_Mobile_Apps/Android/Sample-App-MAS-Android-Access-API-Geolocation-And-OTP/MASAccessAPISample/build.gradle:10]: test comment

**Review by arindamhit — APPROVED** [review]
approved

**Review by arindamhit — APPROVED** [review]
approved

**Review by arindamhit — APPROVED** [review]
approve

**Review by gmthomps — APPROVED** [review]

**Pull request merged**